charts/sdfactory/templates/deployment.yaml

```diff
@@ -107,3 +107,9 @@ spec:
 
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
+          volumeMounts:
+            - mountPath: /tmp
+              name: tmp
+      volumes:
+      - emptyDir: {}
+        name: tmp
```

Comment:
@typecastcloud just to confirm, also for the temurin base image the `tmp` mount is necessary, right?
I mainly have experience with the dotnet and nginx unprivileged base image

Reply:
thanks for confirming